packages/@angular/cli/bin/ng

```diff
@@ -132,9 +133,20 @@ resolve('@angular/cli', { basedir: process.cwd() },
       cli = cli['default'];
     }
 
+    let standardInput;
+    if (process.platform === 'win32') {
```

Comment:
I think this check is an error; don't you want to set it to `process.input` on linux/mac?